recipes/clickhouse-odbc/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "clickhouse-odbc" %}
+{% set version = "1.0.0.20181017" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/yandex/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 73a5ef6ec8ab9b09b45965cec387daed06a4999a84e7f234bb03a78f6accd346
+  patches:
+    # https://github.com/yandex/clickhouse-odbc/pull/150
+    - cac38224ae9ac657c80f56410e701b1509d03333.patch
+    # This patch is already merged in master
+    - 72c8e9acd1a28c501b2bcb2306b8f242b5b73fb6.patch
+
+build:
+  number: 1000
+  skip: true  # [win]
+  skip: true  # [c_compiler == "toolchain_c" and linux]
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - openssl
+    - poco
+    - libtool
+    - unixodbc
+  run:
+    - openssl
+    - poco
+    - unixodbc
+
+test:
+  commands:
+    - test -f $PREFIX/lib/odbc/libclickhouseodbc.dylib  # [unix]
+    - test -f $PREFIX/lib/odbc/libclickhouseodbcw.dylib  # [unix]
+    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
+    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+
+about:
+  home: https://github.com/yandex/clickhouse-odbc
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'ODBC driver for ClickHouse'
+  doc_url: http://clickhouse.yandex/
+  dev_url: https://github.com/yandex/clickhouse-odbc
+
+extra:
+  recipe-maintainers:
+    - kszucs
```

Comment:
@kszucs are you ok with being a maintainer here?

Reply:
Sure! Thanks!